src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/ParenPadCheck.java

```diff
@@ -101,18 +101,78 @@
  * &lt;module name=&quot;ParenPad&quot;/&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * class Foo {
+ *
+ *   public Test(int y ) {  // violation, space before right parenthesis
+ *   }
+ *
+ *   public void foo() {  // OK
+ *     bar( 1);  // violation, space after left parenthesis
+ *   }
+ *
+ *   public void bar(int k ) {  // violation, space before right parenthesis
+ *     while (k &gt; 0) {  // OK
+ *     }
+ *
+ *     Test obj = new Test(k);  // OK
+ *   }
+ *
+ *   public void fun() {  // OK
+ *     switch(y ) {  // violation, space before right parenthesis
```

Comment:
`// violation, space before right parenthesis`
there are 3 example

Reply:
I didn't understand this one. Could you please explain it again?

Reply:
There are 3 examples but one of them includes two violations

Reply:
it is not clear. 3 violations for me look exactly the same

Reply:
I see 3 same example of violations. They all are the same for me

Reply:
I can't quite understand what you mean by "same" here.
First is for constructor definition,
second is for method call,
third is method definition
fourth one is switch statement.

Reply:
![Capture](https://user-images.githubusercontent.com/43749360/79759151-f2eaba80-833b-11ea-80f0-616916bf86b1.PNG)

There are 4 examples